buildpipeline/Core-Setup-Signing-Validation.json

```diff
@@ -121,7 +121,7 @@
       },
       "inputs": {
         "SourceFolder": "$(PB_SourcesDirectory)",
-        "Contents": "*.log\nBin\\SigningValidation\\Logs\\**",
+        "Contents": "*.log\nbin\\SigningValidation\\Logs\\**",
```

Comment:
Is this also lower case? This is an official build definition, I wonder if this changed as well.